### PR TITLE
Task/43 - [BUG] Issue with displaying LaunchScreenActivity on diffrent screens

### PR DESCRIPTION
**Pull request by Zalezny**
**1. Created new 3 layouts for landscape, smaller screens and tablets screens for LaunchScreenActivity, Examples:**

small phone

/\ small screen

tablet

/\ tablet

Pixel 6 landscape

/\ Pixel 6 landscape

Pixel 6 portrait

/\ Pixel 6 portrait

**2. Created missing annotaion about LaunchScreenActivity in AndroidManifest.**

### CODE REVIEW

**Review by przemek19980102 — CHANGES_REQUESTED** [review]
image
That's how it looks on tablets

+ could you try to use max height attributes on imageView in order to get rid of this many layout variations? In order to complete the task we can have only one layout. Ask me for help if you have any questions :)

**Comment by Zalezny** [comment]
Added changes: one xml file, added guidelines etc.
 

### Small screen:

image
image

### Tablet:

image
image

### Phone (Pixel 6):
image
image

File: app/src/main/res/layout/activity_launch_screen.xml [hunk 1]
```diff
@@ -82,13 +83,45 @@
         android:layout_height="wrap_content"
         android:layout_marginStart="@dimen/size_3x"
         android:layout_marginEnd="@dimen/size_3x"
-        android:layout_marginBottom="@dimen/size_10x"
+        android:layout_marginBottom="@dimen/size_7_5x"
         android:enabled="true"
         android:gravity="center"
-        android:padding="14dp"
+        android:padding="@dimen/size_2x"
```
Comment (msarelo) [inline app/src/main/res/layout/activity_launch_screen.xml:89]: In zepplin there is 14dp:
image

On my phone text is cut off from the bottom:
image

**Review by msarelo — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/fragment_launch_screen.xml [hunk 2]
```diff
@@ -13,15 +13,14 @@
...
         android:layout_height="wrap_content"
-        android:layout_marginTop="@dimen/size_5x"
-        android:layout_marginBottom="@dimen/size_3x"
+        android:layout_marginTop="@dimen/size"
+        android:layout_marginEnd="@dimen/size_3x"
         android:fontFamily="@font/poppins_regular"
         android:letterSpacing="0.04"
         android:text="@string/launch_screen_txt_sign_in"
         android:textColor="@color/s_60"
         android:textSize="14sp"
```
Comment (przemek19980102) [inline app/src/main/res/layout/fragment_launch_screen.xml:22]: Don't we have a style for the text already? You shouldn't need to put fontFamily, letterSpacing etc.

**Review by przemek19980102 — CHANGES_REQUESTED** [review]

**Review by forceindia712 — APPROVED** [review]

**Pull request merged**